src/txn/txn_recover.c

```diff
@@ -714,7 +714,7 @@ __recovery_setup_file(WT_RECOVERY *r, const char *uri, const char *config)
     WT_DECL_RET;
     WT_LSN lsn;
     uint32_t fileid, lsnfile, lsnoffset;
-    char lsn_str[WT_MAX_LSN_STRING];
+    char lsn_str[WTI_MAX_LSN_STRING];
```

Comment:
I suppose we are making this WT and moving it to log.h?

Reply:
@Siddhartha8899 Fixed in e8b00b125b5dd093ec54e6322879bf3b598d0d48